share/spice/dota2/dota2_item.handlebars

```diff
@@ -0,0 +1,60 @@
+<div class="gw">
+    <div class="g">
+        <img src="{{image}}" title="{{name}}"/>
+    </div>
+    <div class="g item-info">
+        <h1 class="zci__header">
+            {{name}}
+        </h1>
+        <hr class="divider" />
+        <div class="twenty text--secondary">
+            Cost
+        </div>
+        <div class="eighty eighty--screen-s seventy--screen-xs">
+            <img src="{{assets.gold.img}}" title="{{assets.gold.title}}"/> {{cost}}
```

Comment:
D'oh! I should have caught that. :-P I'll fix that, but I'll wait to see if there are any other changes needed.